lib/permutations.rb

```diff
@@ -1,4 +1,24 @@
-
 def permutations?(string1, string2)
```

Comment:
What about "aaaabb" and "bbbbaa"?

Not quite working.  Maybe count how many of each letter.